front/src/statistics/Statistics.js

```diff
@@ -0,0 +1,310 @@
+    setDayNumber(dayNum);
+
+    const fetchData = async () => {
+      const monthResult = await getMonthlyVisitors(firstDate, lastDate); // eslint-disable-line
+      const dayResult = await getDailyVisitors(singleDay); // eslint-disable-line
+      setDailyUsers(dayResult);
+      setMonthlyUsers(monthResult);
+    };
+    fetchData();
+  }, [date]);
+
+  useEffect(() => {
+    // Count monthly visitors
+    let day = 1;
+    let dayArray = [];
+    monthlyUsers.forEach((user) => { // eslint-disable-line
+      dayArray = dayArray.concat(day);
+      day += 1;
+    });
+    // Options for the month chart
+    setMonthOptions({
+      chart: { id: 'monthChart' },
+      xaxis: {
+        categories: dayArray,
+        title: {
+          text: statistics.DayLabel[fin],
+          style: {
+            fontSize: '14px',
+          },
+        },
+      },
+      yaxis: {
+        title: {
+          text: statistics.VisitorLabel[fin],
+          style: {
+            fontSize: '14px',
+          },
+        },
+      },
+      stroke: {
+        curve: 'smooth',
+        colors: ['#658f60'],
+      },
+    });
+    setMonthSeries([{ name: 'visitors', data: monthlyUsers }]);
+  }, [monthlyUsers]);
+
+  useEffect(() => {
+    // Count daily visitors
+    let track = 1;
+    let visitorArray = [];
+    dailyUsers.forEach((user) => { // eslint-disable-line
+      visitorArray = visitorArray.concat(track);
+      track += 1;
+    });
+    // Options for the day chart
+    setDayOptions({
+      chart: { id: 'dayChart' },
+      xaxis: {
+        categories: visitorArray,
+        title: {
+          text: statistics.TrackLabel[fin],
+          style: {
+            fontSize: '14px',
+          },
+        },
+      },
+      yaxis: {
+        title: {
+          text: statistics.VisitorLabel[fin],
+          style: {
+            fontSize: '14px',
+          },
+        },
+      },
+      fill: {
+        colors: ['#658f60'],
+      },
+      dataLabels: {
+        enabled: true,
+        style: {
+          fontSize: '16px',
+        },
+        background: {
+          enabled: true,
+          foreColor: '#000000',
+        },
+      },
+    });
+    setDaySeries([{
+      name: 'visitors',
+      data: dailyUsers,
+    }]);
+  }, [dailyUsers]);
+
+  const previousDayClick = () => {
+    const newDate = new Date(date.setDate(date.getDate() - 1));
+    setDate(newDate);
+  };
+
+  const nextDayClick = () => {
+    const newDate = new Date(date.setDate(date.getDate() + 1));
+    setDate(newDate);
+  };
+
+  const handleDateChange = (newDate) => {
+    setDate(newDate);
+  };
+
+  const continueWithDate = (event) => {
+    if (event !== undefined && event.type !== undefined && event.type === 'submit') {
+      event.preventDefault();
+    }
+  };
+
+  const countMonthlyVisitors = () => {
+    let total = 0;
+    monthlyUsers.forEach((day) => {
+      total += day;
+    });
+    return total;
```

Comment:
Use

return monthlyUsers.reduce((a, b) => return a + b, 0)

to get a sum of a num array. The syntax may not be absolutely correct but you get the idea